recipes/ucc/meta.yaml

```diff
@@ -0,0 +1,50 @@
+package:
+  name: ucc
+  version: v1.2.0
+
+source:
+  url: https://github.com/openucx/ucc/archive/refs/tags/v1.2.0.tar.gz
+  sha256: c1552797600835c0cf401b82dc89c4d27d5717f4fb805d41daca8e19f65e509d
+  patches:
+    - 0001-EC-CUDA-fix-build-with-12.2-and-older-arch-847.patch  # [(cuda_compiler_version or "None").startswith("12")]
+
+build:
+  skip: True  # [not linux]
+  skip: True  # [cuda_compiler_version == "None"]
+  skip: True  # [cuda_compiler_version == "11.2"]
+  number: 0
+  missing_dso_whitelist:
+    - '*libcuda.so*'
+  ignore_run_exports_from:
+    # CXX standard library is unused
+    - {{ compiler('cxx') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+    - automake
+    - libtool
+    - make
+
+  host:
+    - ucx
+    - cuda-version {{ cuda_compiler_version }}
+    - cuda-cudart-dev  # [(cuda_compiler_version or "None").startswith("12")]
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libucc$SHLIB_EXT
```

Comment:
Please use this section to also check for the other artifacts that are being shipped: cmake files, headers, and binary.

Reply:
Check the versioned library instead of the versionless link to prevent ABI breaks in future packages.

Reply:
Done.